pkg/cmd/ssh/options.go

```diff
@@ -151,6 +151,103 @@ var (
 
 		return cmd.Run()
 	}
+	// waitForSignal informs the user about their SSHOptions and keeps the
+	// bastion alive until gardenctl exits.
+	waitForSignal = func(ctx context.Context, o *SSHOptions, shootClient client.Client, bastion *operationsv1alpha1.Bastion, nodeHostname string, nodePrivateKeyFiles []string, signalChan <-chan struct{}) error {
```

Comment:
Seeee my comment in the other ssh PR.

Reply:
as just discussed we leave it as is for now until we get feedback from users that it is confusing when targeting managed seed (for which there is always a shoot) and running ssh